.github/workflows/Dev-Promotion.yml

```diff
@@ -0,0 +1,100 @@
+---
+name: Dev promotion
+
+on: 
+#  push:
+#    branches: 
+#      - master
+#    paths-ignore:
+#      - '.github/workflows/**'
+#      - 'test-script'
+#      - '**/pom.xml' 
+  workflow_dispatch:
+
+env:
+  IMAGE_BASE_NAME: ceres-test
+  HELM_CHART: ceres
+  HELM_CHART_FOLDER: helm-ceres/cherts/ceres
+  CLUSTER_NAME: primary-cluster
+  REGISTRY: gcr.io
+
+jobs:
+  Build:
+    runs-on: ubuntu-latest
+    outputs:
+      version: ${{ steps.setVersion.outputs.version }}
+    steps:
+    ### Seting up runner env ###
+      - uses: actions/checkout@v3
+
+      - uses: ./.github/actions/GCPSetup
+        with:
+          GCPCreds: ${{ secrets.GCP_SA_JSON }} 
+
+      - name: 'Maven settings'
+        run: 'gsutil cp gs://salus-mavenrepository/m2-settings.xml ./.mvn/settings.xml'
+
+      - name: Docker login
+        run: gcloud auth configure-docker ${{ env.REGISTRY }} -q
+
+      - name: .m2 forder [*** fix it please ***]
+        run: sed -i '/<localRepository>/d' ./.mvn/settings.xml
+
+      - name: Set up JDK
+        uses: actions/setup-java@v3
+        with:
+          java-version: '14'
+          distribution: 'adopt-hotspot'
+          cache: maven
+
+      - name: Set version
+        id: setVersion
+        shell: bash
+        run: |
+          POM_VERSION=$(mvn help:evaluate -Dexpression=project.version -q -DforceStdout)
```

Comment:
This will have a `-SNAPSHOT` suffix on it. Try adding something like `POM_VERSION="${POM_VERSION%-SNAPSHOT}"` to get rid of it.